overlord/devicestate/devicestate.go

```diff
@@ -320,8 +338,9 @@ func Remodel(st *state.State, new *asserts.Model) ([]*state.TaskSet, error) {
 	if current.Series() != new.Series() {
 		return nil, fmt.Errorf("cannot remodel to different series yet")
 	}
-	// FIXME: we need language in the model assertion to declare what
-	// transitions are ok before we allow remodel like this.
+	// FIXME: we need language in the model assertion to declare
```

Comment:
// TODO: we need dedicated assertion language to permit for model transitions  before we allow that cross vault